examples/interop/pos-mainchain-fast/src/commands/start.ts

```diff
@@ -134,7 +134,7 @@ export class StartCommand extends BaseStartCommand {
 			app.registerPlugin(new DashboardPlugin(), { loadAsChildProcess: true });
 		}
 		if (flags['enable-chain-connector-plugin']) {
-			app.registerPlugin(new ChainConnectorPlugin(), { loadAsChildProcess: true });
+			app.registerPlugin(new ChainConnectorPlugin(), { loadAsChildProcess: false });
```

Comment:
why loading it in memory?

Reply:
ah i did that due to this issue https://github.com/LiskHQ/lisk-sdk/issues/8629 locally but maybe we can leave it inMemory here and later we can change it if we want when the issue is fixed